tests/utils/useUpdateAtom.test.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { StrictMode, useEffect, useRef } from 'react'
+import { fireEvent, render } from '@testing-library/react'
+import { Provider, atom, useAtom, WritableAtom } from '../../src/index'
+import { useUpdateAtom } from '../../src/utils'
+
+const useRerenderCount = () => {
+  const rerenderCountRef = useRef(0)
+  useEffect(() => {
+    rerenderCountRef.current += 1
```

Comment:
I called this commit counts or `commits` in other files, but let's revisit it later because I don't know which is better.

Reply:
I see, I agree, it would be better to be completely consistent.